study.md

```diff
@@ -34,13 +34,14 @@ In ActiveRecord Migrations, what is the name of the method the creates a new
 table?
 
 ```md
-<!-- your response here -->
+
+$ bin/rails generate migration CreateX
```

Comment:
We want the Rails method that's used to create a new table and it's `create_table`.